cl/sentinel/handlers/handlers.go

```diff
@@ -29,15 +32,39 @@ import (
 	"github.com/libp2p/go-libp2p/core/protocol"
 )
 
+type RateLimiter struct {
+	counter         uint32
+	lastRequestTime time.Time
+	limited         bool
+}
+
+type RateLimits struct {
+	pingLimit       uint32
+	goodbyeLimit    uint32
+	metadataV1Limit uint32
+	metadataV2Limit uint32
+	statusLimit     uint32
+}
+
+const punishmentPeriod = 15 //mins
+
+var defaultRateLimits = RateLimits{
+	pingLimit:       5000,
+	goodbyeLimit:    5000,
+	metadataV1Limit: 5000,
+	metadataV2Limit: 5000,
+	statusLimit:     5000,
+}
+
 type ConsensusHandlers struct {
-	handlers      map[protocol.ID]network.StreamHandler
-	host          host.Host
-	metadata      *cltypes.Metadata
-	beaconConfig  *clparams.BeaconChainConfig
-	genesisConfig *clparams.GenesisConfig
-	ctx           context.Context
-
-	beaconDB persistence.RawBeaconBlockChain
+	handlers       map[protocol.ID]network.StreamHandler
```

Comment:
try running `make lint` here, there are extra spaces here, this does not look good. i think `gofmt -w -s .` should get rid of those, VScode also have an extension to format go files.

Reply:
Fixed.